README.md

```diff
@@ -1,38 +1,65 @@
 
 # Material_FAB
 
-Material FAB replicates the floating action buttons(FAB) using *material design 3* specifications.
+Material FAB replicates the floating action buttons(FAB) using *material design 3* specifications in openHarmony.
+
+## Dependencies
+For using Material_FAB in your app, add the below dependency in entry/package.json
+
+```
+"dependencies": {
+    "@ohos/Material_FAB": "file:../Material_FAB"
+  }
+```
+
+## Usage Instructions
+
+Import all components at once
+
+```ets
+import { FAB_regular, FAB_extended }  from '@ohos/Material_FAB'
+```
 
-## <FAB_regular>
 
-Implementation of Regular FAB buttons with size options as *small*, *medium* and *large*.
 ## Screenshots
 
+![img.png](images/sample_small_fab.png)   ![img.png](images/sample_medium_fab.png)
+![img.png](images/sample_large_fab.png)   ![img.png](images/sample_extended_fab.png)
 
+## <FAB_regular>
 
+Implementation of Regular FAB buttons with size options as *small*, *medium* and *large*.
 
-## Attributes
+### Attributes
 
+#### Icon properties
+| Attribute | Type     | Description                |
+| :-------- | :------- | :------------------------- |
+| `fabIcon` | `ResourceStr` | specifies the resource for the icon. Defaults to `+` icon. |
+| `iconSize` | `SizeOptions` | Give custom size to the icon. |
 
+#### Container properties
 | Attribute | Type     | Description                |
 | :-------- | :------- | :------------------------- |
-| `fabSize` | `string` | specifies the type of FAB_regular from small, medium(**DEFAULT**) and large  |
-| `fabIcon` | `ResourceStr` | specifies the resource for the icon (Default - Plus icon) |
-| `iconSize` | `SizeOptions` | Give custom size to the icon |
-| `position` | `Position` | Specify the position of the FAB (Default [0, 0]) |
-| `backgroundColor` | `ResourceColor` | Specify the background color for the FAB (Default - Blue) |
-| `border` | `BorderOptions` | give custom border to the FAB |
-| `fabCustomSize` | `SizeOptions` | Give custom size for the FAB |
-| `margin` | `Margin` | Give margin to the FAB |
-| `rippleColor` | `ResourceColor` | Give custom size color for the ripple effect (Default - White) |
-| `onClick` | `callback` | Give behaviour for the click event |
+| `fabSize` | `string` | specifies the type of FAB_regular from small, medium and large. Defaults to *medium*.  |
+| `position` | `Position` | Specify the position of the FAB. Defaults to [0, 0]. |
+| `backgroundColor` | `ResourceColor` | Specify the background color for the FAB. Defaults to *Blue*. |
+| `border` | `BorderOptions` | give custom border to the FAB. |
+| `fabCustomSize` | `SizeOptions` | Give custom size for the FAB. |
+| `margin` | `Margin` | Give margin to the FAB. |
 
 
+#### Other properties
```

Comment:
Remove the test report from the project repo. It will be added as an artifact during github release anyways.